src/Backend/HPTA.Api/appsettings.json

```diff
@@ -23,5 +23,8 @@
   "MicrosoftGraph": {
     "BaseUrl": "https://graph.microsoft.com/v1.0",
     "Scopes": "user.read"
+  },
+  "AppSettings": {
```

Comment:
It is better to name the property as "ConnectionStrings" instead of "AppSettings". We will add more properties into the app settings and will use Options pattern to access them.